src/hisssql.py

```diff
@@ -33,6 +35,12 @@ def main():
 
     new_src = Orchestrator.transform(src)
 
+    # Take care of linting with autoflake
+    # TODO: This does not exactly work as expected, as the requirements for 'unused variables' seem a little weird
```

Comment:
Do you guys know a linter with an API, that is actually able to remove dead code? Otherwise we might need to be able to flag statements in the original src_code for deletion...